test/typecheck/main.go

```diff
@@ -401,7 +400,7 @@ func main() {
 				if len(dir) > 80 {
 					dir = dir[:80]
 				}
-				fmt.Printf("\r%d/%d \033[2m%-13s\033[0m %-80s", proc, total, platform, dir)
+				fmt.Printf("\r%d/%d \033[2m%-13s\033[0m %-80s\n", proc, total, platform, dir)
```

Comment:
[1]
i don't think these 3 changes are needed because the loops are supposed to append on the same line.
the other changes seem fine.

Reply:
has been updated.